### ISSUE TITLE
Better Flow Documentation

### ISSUE THREAD

**Issue by pzuraq** [paddle8/ember-popup-menu#10]
There doesn't appear to be any documentation of the concept behind Flows, currently trying to figure out how they work. I'll make a PR once I have a better understanding.

**Comment by tim-evans** [comment]
@pzuraq I'll try to document some of it. If you could take a second pass, that'd be amazing :+1:

**Comment by tim-evans** [comment]
@pzuraq If you could fiddle with https://github.com/paddle8/ember-popup-menu#flows, I'd greatly appreciate it.

**Comment by pzuraq** [comment]
@tim-evans: Looks pretty solid, I'm still not fully grasping what `andSlideBetween` does though. I'm trying to setup an example, maybe including one would be a good idea?

**Comment by tim-evans** [comment]
Yeah, that's a bit trickier to show. A gif would probably be best
